rocAL/rocAL/source/rali_api_augmentation.cpp

```diff
@@ -60,11 +60,48 @@ THE SOFTWARE.
 #include "meta_node_rotate.h"
 #include "meta_node_ssd_random_crop.h"
 #include "meta_node_flip.h"
+#include "node_sequence_rearrange.h"
 
 #include "commons.h"
 #include "context.h"
 #include "rali_api.h"
 
+RaliImage  RALI_API_CALL
+raliSequenceRearrange(
+            RaliContext p_context,
+            RaliImage p_input,
+            unsigned int* new_order,
+            unsigned int  new_sequence_length,
+            unsigned int sequence_length,
+            bool is_output )
+{
+    if(!p_context)
+        THROW("Null values passed as input")
+    if(!p_input)
+        THROW("Null values passed as input")
+    Image* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    try
+    {
+        if(sequence_length == 0)
+            THROW("sequence_length passed should be bigger than 0")
+        auto input = static_cast<Image*>(p_input);
+        auto info = ImageInfo(input->info().width(), input->info().height_single(),
+                              context->master_graph->internal_batch_size() * new_sequence_length,
+                              input->info().color_plane_count(),
```

Comment:
please follow the change is tot master to get rid of all codacy errors regarding !p_context

Reply:
Have added the changes to fix codacy errors.